### ISSUE TITLE
New icon 'basket'

### ISSUE THREAD

**Issue by UteV** [uikit/uikit#3340]
Please add a shopping basket icon.

**Comment by guiguidu60** [comment]
It already exists: cart.
It's almost the same thing.

**Comment by UteV** [comment]
Not quite. While a 'basket' icon merely signifies that items can be collected in a specified place for further processing, the 'cart' icon means shopping functionality, which is something our site does not offer.

**Comment by guiguidu60** [comment]
Anyway, it lacks many (essential) icons.

**Comment by rangka-kacang** [comment]
Hi.

I don't think there's a need to bloat the icon js. Not everyone needs all the things you want for their projects. While, UIKit3 is good enough as a starter framework, you may want to extend it to suite your project specifically. For example, if you need a "basket" icon, why not head over to [icomoon app](https://icomoon.io/app/)? You'll have plenty of choices there and use exactly what you need and minimize your file size in general. P/s: I don't endorse nor promoting the app, it somehow just related to this question as an alternative workaround.

I never like the UIKit3 icons anyway, they are too light for my taste and the positioning isn't quite well arrange (in comparison to FA5 svg js).

Thank you.

**Comment by haydnjames** [comment]
Personally I think there are already too many icons. Who uses UIKit without customising it anyways? So there only really needs to be a handful of icons to show you how to customise - then the only limit is your creativity and imagination.

**Comment by jsieler** [comment]
Why not just to use fontawesome?
